misc/formatter/modules/formatter-core/src/test/resources/declarations/module-enumeration/assert/module_enumeration_declaration_1.bal

```diff
@@ -5,8 +5,7 @@ public enum Color {
 }
 
 public enum Language {
-    EN = "English",
-    TA = "Tamil"
+    EN = "English", TA = "Tamil"
```

Comment:
Is this intentionally changed to inline?

Reply:
Yup,
```ballerina
public   enum   Language  {
    EN   =   "English"  ,       TA   = "Tamil"
}
```
actually, the PR fixes when the enum members are inline, the formatting should also support that. Earlier this wasn't the case, thus, assert case is changed